### DISCUSSION TITLE
/v2/sp/targets/bidRecommendations

### DISCUSSION THREAD

**Discussion by Lin1031** [amzn/ads-advanced-tools-docs#119] [Q&A] [answered]
Hello, 
I'm trying to get bid recommendations for the IT country using the /v2/sp/targets/bidRecommendations API. 
However, I've noticed that the data I'm getting back doesn't match up with what's in the Amazon backend. 
When I call /sp/targets/bid/recommendations on the US site, I get results that match up with the backend.
I'm wondering when other sites, besides the ones currently supported (US, UK, DE, CA, JP, IN, ES, and FR), will be able to use the v3 API.

**Comment by elizafarley** [comment, marked as answer]
Hi @Lin1031, If you think there might be a bug with mismatched data /v2/sp/targets/bidRecommendations, please open a [Jira ticket](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2/create/6) so the support team can look into it.

Regarding expanded availability for /sp/targets/bid/recommendations, I can't provide a date for that at this time, but please continue to check our [release notes](https://advertising.amazon.com/API/docs/en-us/release-notes/index). If it does become available in other regions, I'll also update you here.

**Reply by Lin1031** [reply]
Thank you for your response. Our developers and business team have double-checked the data once again, and it appears that the backend data and the data returned by the requests are consistent. It was just a misunderstanding. We are checking the release notes every day and looking forward to your updates.
